UPDATING.md

```diff
@@ -409,7 +409,7 @@ The following table shows changes in import paths.
 |-------------------------------------|----------------------------|
 | airflow.hooks.base_hook.BaseHook | airflow.hooks.base.BaseHook |
 | airflow.hooks.dbapi_hook.DbApiHook | airflow.hooks.dbapi.DbApiHook |
-| airflow.operators.dummy_operator.DummyOperator | airflow.operators.dummy.DummyOperator |
+| airflow.operators.dummy_operator.NoOpOperator| airflow.operators.dummy.NoOpOperator|
```

Comment:
We should keep it, because it is related to released version.

Reply:
Yeah. We should only deprecate it. and also we should chnage package name as well.